packages/mantine/package.json

```diff
@@ -0,0 +1,89 @@
+{
+    "name": "@coveord/plasma-mantine",
+    "version": "0.0.0",
```

Comment:
Not sure what version it should initially have 🤔

Reply:
I would start it at v1.0.0 I think? It was working in the admin so it should be mature enough to be a major? 🤔

Reply:
It will automatically bump to 47 or whatever when we merge the PR

Reply:
<img src="https://media4.giphy.com/media/JU43PP6U93ij6kVTgj/giphy.gif"/>